doc/audio.md

```diff
@@ -13,75 +22,75 @@ flutter:
     - assets/audio/music.mp3
```

Comment:
I don't know if I missed, and if I didn't where to put it, but we should probably mention that the default prefix that is set by flame_audio is `/assets/audio` (which can be changed) and audio_pool I believe might use `/assets/audio/sfx` (not 100% sure)

Reply:
Verified it, its correct. I'll add a sentence about that.